src/Rules/AustrianInsuranceNumber.php

```diff
@@ -51,6 +52,25 @@ private function startsNotWithZero(string $svnumber): bool
         return (int) $svnumber[0] !== 0;
     }
 
+    private function hasValidBirthday(string $svnumber): bool
+    {
+        $splittedBirthday = str_split(substr($svnumber, 4, 10), 2);
```

Comment:
From the [specification](https://www.sozialversicherung.at/cdscontent/?contentid=10007.820902&viewmode=content#datareplacementdoneWorausbestehtdieVersicherungsnummer820902) I can see that the date of birth is in the 5-10 digit. The `substr` call, on the other hand, fetches the digits 5-15 `$offset` to `$offset` + `$length`. This is irrelevant for the 10-digit number, as it is always read to the end. This means that the length can also be omitted directly.

```suggestion
        $splittedBirthday = str_split(substr($svnumber, 4), 2);
```

Reply:
Thanks!